src/main/java/fileservice/FileService.java

```diff
@@ -0,0 +1,9 @@
+package fileservice;
+
+import java.util.List;
+
+public interface FileService {
+    List read(String file);
+
+    void write(String file, String content);
```

Comment:
separate read and write operations into two services. Because you may need to read from csv but write to json, for example